### ISSUE TITLE
How to reproduce tokenization for CJK NNLM models?

### ISSUE THREAD

**Issue by chezou** [tensorflow/hub#181]
As far as I read documentation on tfhub.dev, we need to tokenize word before using pre-trained text embedding model like [nnlm-ja-dim128](https://tfhub.dev/google/nnlm-ja-dim128/1), [nnlm-ko-dim128](https://tfhub.dev/google/nnlm-ko-dim128/1), [nnlm-zh-dim128](https://tfhub.dev/google/nnlm-zh-dim128/1).

To prevent tokenization mismatch between pre-trained models and new sentences to be executed, and to reduce number of OOV, we should use same tokenization and/or normalization method (such as [NFKC conversion](https://en.wikipedia.org/wiki/Unicode_equivalence#Normal_forms), [example normalization in mecab-ipadic-neologd](https://github.com/neologd/mecab-ipadic-neologd/wiki/Regexp#python-written-by-hideaki-t--overlast)).

Is there any code or document for reproduce tokenization/normalization?

**Comment by chezou** [comment]
Close since this isn't a bug. Will ask on the ML.
